config/linters/reek.yml

```diff
@@ -5,7 +5,9 @@ PrimaDonnaMethod:
 NestedIterators:
   max_allowed_nesting: 2
 UncommunicativeMethodName:
-  enabled: false
+  enabled: true
+  accept:
+    - google_oauth2
```

Comment:
@sinpai You can change `google_oauth2` to another name, for example: `google`. 

[See here](https://github.com/zquestz/omniauth-google-oauth2#configuration), `name` option.